Makefile

```diff
@@ -40,15 +40,15 @@ test:
 ## it-stress: Runs Integration Tests stress mode
 it-stress: build
 	@echo "  >  \033[32mRunning stress tests...\033[0m "
-	MODE=stress go test ./tests/stress/... -timeout=15m -v -run TestSync_
+	MODE=stress go test ./tests/stress/... -timeout=25m -v -run TestSync_
```

Comment:
Why increase timeout? Do tests take 10 minutes longer now?

Reply:
Yeah, the tests have been timing out. I've just been playing around with these values. Not sure what we wanna use yet.

Reply:
ah okay I see, interesting. Will be curious to see where a good number is.

Do you have any idea why it's slower with these changes?

Reply:
reverted